packages/components-angular/projects/consumer-app/cypress/e2e/components.cy.ts

```diff
@@ -1,24 +1,40 @@
-import * as Components from '@swisspost/design-system-components/loader';
-
-const COMPONENT_TAG_NAMES = Object.keys(Components)
-  .filter(c => /^Post([A-Z][a-z]+)+$/.test(c))
-  .map(c => c.replace(/([a-z0–9])([A-Z])/g, '$1-$2').toLowerCase());
+import { setupComponentErrorCapture, assertNoComponentErrors } from '../support/component-error-filter';
+import { componentNames } from '@swisspost/design-system-components/dist/component-names.json';
 
 describe('Components', () => {
   beforeEach(() => {
     cy.visit('/');
-    cy.window().then(win => {
-      cy.wrap(cy.spy(win.console, 'error')).as('consoleError');
+  });
+
+  componentNames.forEach(componentName => {
+    it(`should contain <${componentName}>`, () => {
+      cy.get('body').then($body => {
+        if ($body.find(componentName).length > 0) {
+          cy.get(componentName).first().should('exist');
+        }
+      });
     });
   });
 
-  it('should not log any error', () => {
-    cy.get('@consoleError').should('not.have.been.called');
+  componentNames.forEach(componentName => {
+    it(`should render and be attached: <${componentName}>`, () => {
+      cy.get('body').then($body => {
+        if ($body.find(componentName).length > 0) {
+          cy.get(componentName).first().should('exist');
+        }
+      });
+    });
```

Comment:
This is basically the same as the test before.
If you want to test attachement, you could maybe test if `${componentName}[data-hydrated]` exists.